plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/containers/UnityCatalogContainer.java

```diff
@@ -184,6 +194,34 @@ public void execute(@Language("SQL") String sql)
     public void close()
             throws Exception
     {
+        queryRunner.close();
         closer.close();
     }
+
+    public void dropTable(String schema, String tableName)
+    {
+        Request request = Request.Builder.prepareDelete()
+                .setUri(URI.create(uri() + "/tables/%s.%s.%s?catalog_name=%s&schema_name=%s".formatted(catalogName, schema, tableName, catalogName, schema)))
+                .build();
+        execute(request);
+        queryRunner.execute("DROP TABLE iceberg.tpch.%s".formatted(tableName));
```

Comment:
Why do we need 2 calls (HTTP request & DROP TABLE)? Can we leave a code comment?